src/mobiflight.cpp

```diff
@@ -229,8 +243,8 @@ void setup()
   // Time Gap between Inputs, do not read at the same loop
   lastAnalogAverage = millis() + 4;
   lastAnalogRead = millis() + 4;
-  lastButtonUpdate= millis();
-  lastEncoderUpdate = millis() +2;
+  lastButtonUpdate = millis();
+  lastEncoderUpdate = millis() + 2;
```

Comment:
Could you add:
`lastInputShifterUpdate = millis() + 6;`
to also separate the input loop from the other input devcices. And it must encapsulated (#if MF_INPUTSHIFTER_SUPPORT) what I forget for then Analog input.

Reply:
Good catch, thanks! I added the analog #if too.